src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -10973,6 +10986,12 @@ .locals init (delegate*<ref System.Span<int>, ref System.Span<int>> V_0)
   IL_0009:  ret
 }
 ");
+
+            comp = CreateCompilationWithSpan(source, options: TestOptions.UnsafeReleaseExe);
+            comp.VerifyDiagnostics(
+                // (14,62): error CS8166: Cannot return a parameter by reference 'i' because it is not a ref parameter
```

Comment:
We already have an issue for improving this diagnostic, right?

Reply:
Yes, [https://github.com/dotnet/roslyn/issues/62768](https://github.com/dotnet/roslyn/issues/62768).